### ISSUE TITLE
About fine tuning discriminator

### ISSUE THREAD

**Issue by sourabh-patil** [amanchadha/iSeeBetter#31]
Hi there! You said you fine-tuned the pre-trained RBPN for 4 epochs. I was curious about the values of Generator and Discriminator loss and their behavior. Do they follow the SRGAN? Also, how did you decide to fine-tune it for 4 epochs? By observing losses and evaluation set scores?
